platform/platform-filter-delegate/src/test/java/org/codice/ddf/platform/filter/delegate/DelegateServletFilterTest.java

```diff
@@ -329,6 +327,11 @@ public void testRemoveSecurityFilter() throws IOException, ServletException {
     }
 
     @Test
+    public void testRemoveNullSecurityFilter() throws IOException, ServletException {
+        delegateServletFilter.removeSecurityFilter(null);
```

Comment:
I don't really care if those methods are called or not, as long as no exception is thrown.